app/assets/stylesheets/application.css

```diff
@@ -1 +1,26 @@
 /* Application styles */
+
+.pcr-app{
+    position: static !important;
+    background: none !important;
+    box-shadow: none !important;
+    padding: 0 !important;
+    width: 100% !important;
+}
+.pcr-color-palette{
+    height: 12em !important;
+    width: 21.5rem !important;
+}
+.pcr-palette{
+    border-radius: 10px !important;
+}
+.pcr-palette:before{
+    border-radius: 10px !important;
+}
+.pcr-color-chooser{
+    height: 1.5em !important;
+}
+.pcr-picker{
+    height: 20px !important;
+    width: 20px !important;
+}
```

Comment:
We've made some recent changes to the Tailwind files, so you'll want to pull those in first.  After that, we should move this to the `@layer components` directive within the `tailwind/application.css` file.